### PR TITLE
TST: fix test_reslogit.py

### PR DESCRIPTION
**Pull request by julesdesir**
## Description of the goal of the PR

Description: fix hyperparameters values to make the tests of test_reslogit.py pass

## Changes this PR introduces (fill it before implementation)

- [x] : Increase dataset size
- [x] : Increase learning rate
- [x] : Increase batch size
- [x] : Switch from SGD optimizer to Adam

## Checklist before requesting a review
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have typed my code
- [x] I have created / updated the docstrings
- [ ] I have updated the README, if relevant
- [ ] I have updated the requirements files if a new package is used
- [x] I have tested my code
- [x] The CI pipeline passes
- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by julesdesir** [comment]
I ran all the tests with PyTest 5 times locally and it successfully passed every times. Fingers crossed!

**Review by VincentAuriau — APPROVED** [review]
🤞🤞🤞
